### ISSUE TITLE
Rainloop not use the php tmp directory ("upload_tmp_dir" config value) but "/tmp" on file upload

### ISSUE THREAD

**Issue by March-M** [RainLoop/rainloop-webmail#1335]
**RainLoop version, browser, OS:**
Rainloop 1.10.5.192, Chrome latest, mac os 10.11.6
Server os: Debian 8.4
Apache version: Apache/2.4.10 (Debian)
php : PHP 5.6.29-0+deb8u1 via php5-fpm module

**Expected behavior and actual behavior:**

The server have the tmp directory value  set to /var/www/mail.mysite.tld/php_tmp, correctly set and correctly work (tested with a simple upload script on the mail.mysite.tld vhosts, it work), but when i try to upload files on mail response, all y get is a "temporary file missing" and a error on my log. Rainloop try to access to /tmp, but why if php use a other temporary directory?
If if modify my config to allow access to /tmp, it correctly work, but it's not the intended behavior, rainloop should use the tmp directory indicated on the php configuration ( ini_get('upload_tmp_dir') )

**Logs or screenshots:**
`[Wed Feb 08 10:55:35.379206 2017] [:error] [pid 22938] [client 109.190.186.230:57531] FastCGI: server "/usr/lib/cgi-bin/php5-fcgi-mail.mysite.tld" stderr: PHP message: PHP Warning:  Unknown: open_basedir restriction in effect. File(/tmp) is not within the allowed path(s): (/var/www/mail.mysite.tld/public_html:/var/www/mail.mysite.tld/php_tmp:/var/www/mail.mysite.tld/php_session`

**Comment by RainLoop** [comment]
Interesting, but RainLoop don't do anything special with this configuration.

**Comment by March-M** [comment]
Exactly, I think the problem comes from this: rainloop should use the value of the php configuration for temporary files, instead of directly using the /tmp from linux. I think the problem comes from this, I checked my configuration by consciousness and everything is OK, no reference to /tmp , That's why I started to think that the little bug came from rainloop and made the issue.

But it is true that it is strange, a normal file upload does not need to know the temporary folder, it retrieves it alone from the config normally, in php.

**Comment by caffeinelucy** [comment]
> 
> 
> Exactly, I think the problem comes from this: rainloop should use the value of the php configuration for temporary files, instead of directly using the /tmp from linux. I think the problem comes from this, I checked my configuration by consciousness and everything is OK, no reference to /tmp , That's why I started to think that the little bug came from rainloop and made the issue.
> 
> But it is true that it is strange, a normal file upload does not need to know the temporary folder, it retrieves it alone from the config normally, in php.

has this ever been solved? we have the same issue and no clue how to fix it so far

**Comment by caffeinelucy** [comment]
> 
> 
> > Exactly, I think the problem comes from this: rainloop should use the value of the php configuration for temporary files, instead of directly using the /tmp from linux. I think the problem comes from this, I checked my configuration by consciousness and everything is OK, no reference to /tmp , That's why I started to think that the little bug came from rainloop and made the issue.
> > But it is true that it is strange, a normal file upload does not need to know the temporary folder, it retrieves it alone from the config normally, in php.
> 
> has this ever been solved? we have the same issue and no clue how to fix it so far

for anyone looking for how to solve this: setting "upload_tmp_dir = /tmp" in the php.ini file fixed it.
